css/_variables.scss

```diff
@@ -212,13 +212,10 @@ $welcomePageTabContentDisplay: inherit;
 $welcomePageTabButtonsDisplay: flex;
 $welcomePageTabDisplay: block;
 
-$welcomePageButtonWidth: 51px;
```

Comment:
These variables are actually being used on the branding repo, although they could probably use a refactor. I'd suggest leaving them in as `inherit` or something that doesn't directly affect the look of the page for the main repo, what do you think @saghul @vp8x8 ?

Reply:
Yes, that's a good idea. `welcomePageButtonWidth` should be kept as `auto` so we can customise it.

Reply:
For me it's fine with leaving that option, still I am not quite sure if that is the right place to leave `$welcomePageButtonWidth: inherit`. IMHO this should not be grouped with other variables used for that `GO` button.

Reply:
Actually `welcomePageButton` is not much clear- it's a bit ambiguous as there are other buttons on the welcome page 😓 I'm going to think a solution for that.